tables/table.py

```diff
@@ -1586,7 +1648,7 @@ def get_where_list(self, condition, condvars=None, sort=False,
             coords = np.sort(coords)
         return internal_to_flavor(coords, self.flavor)
 
-    def itersequence(self, sequence):
+    def itersequence(self, sequence: Sequence) -> Iterator:
```

Comment:
See the comment on `__iter__`

Reply:
As recommended changed to `Iterator[tableextension.Row]`.